tools/health/monitorUtil.py

```diff
@@ -46,8 +46,9 @@ def run(cmd, inData="") :
         p = subprocess.Popen(cmd, stdout=subprocess.PIPE, stdin=subprocess.PIPE, stderr=subprocess.STDOUT)
         if (inData != ""):
             p.stdin.write(inData)
-        p.wait()
```

Comment:
I think we can just remove this file. A quick search through the repo didn't find any reference to it any more. Can you confirm and remove?